packages/perseus-editor/src/hint-editor.tsx

```diff
@@ -89,12 +89,12 @@ export class HintEditor extends React.Component<HintEditorProps> {
         return this.editor.current?.getSaveWarnings();
     };
 
-    serialize: (options?: any) => any = (options: any) => {
+    serialize(options?: any): Hint {
```

Comment:
Refactoring `serialize` to a method instead of an arrow function changes semantics - `this` might be unbound.

Reply:
I've checked that this doesn't happen. 👍